packages/addon/src/utils.ts

```diff
@@ -1,11 +1,15 @@
 import { Cookie } from './types';
 
-export function setCookie(name: string, value: unknown) {
-  document.cookie = `${name}=${JSON.stringify(value)}`;
+export function setCookie(name: string, value: string) {
+  if (typeof value !== 'string') {
```

Comment:
added this to prevent breaking this for existing users who are setting cookie values to objects